src/components/Creatable/Creatable.js

```diff
@@ -238,9 +241,9 @@ function CreatableComponent({
             key={`${item}-${index}`}
             disabled={disabled}
             type={tagType}
-            onClick={() => removeValue(item)}
-            onKeyDown={(e) => isAccessibleKeyDownEvent(e) && removeValue(item)}
-            filter
+            onClick={nonDeletable ? undefined : () => removeValue(item)}
```

Comment:
@BenjaminRuby just a confirmation on the functionality. I think the items initially set (e.g. items already saved in the services) shouldn't be deletable when nonDeletable is true, but items being created by the user before saving should be deletable so the user can edit them right?

Reply:
I added an option to define the initialValues externally, I think this should solve this issue